firebase-firestore/mapping.txt

```diff
@@ -1282,10 +1282,10 @@ com.google.firebase.firestore.local.LocalSerializer -> com.google.firebase.fires
     2101:2103:com.google.firebase.firestore.model.MaybeDocument decodeMaybeDocument(com.google.firebase.firestore.proto.MaybeDocument):55 -> zza
```

Comment:
I think this file is no longer used/maintained. Let's find out if we need it, but if we don't it is probably better to delete this file rather than to change the ProGuard mappings used in the past.

Update: Vlad says it is ok to remove it.